### ISSUE TITLE
Evaluating video-to-labels & labels-to-video translation

### ISSUE THREAD

**Issue by shadow2496** [wangkaihong/Unsup_Recycle_GAN#10]
Hi, I'm leaving an issue to clarify some details on evaluating the video-to-labels and labels-to-video experiments.

When I run the command below following the README, label predictions are generated as RGB images.
```
python test.py --dataroot path/to/data/ --model unsup_single --dataset_mode unaligned_scale --name v2l_experiment --loadSizeW 256 --loadSizeH 256 --resize_mode rectangle --fineSizeW 256 --fineSizeH 256 --crop_mode none --which_model_netG resnet_6blocks --no_dropout --which_epoch 5
```
To calculate MP, AC, and mIoU as in the paper, the generated labels' format should be converted from RGB into actual labels. Could you describe how you did the quantization of the labels? It would also be great if you could provide the code for the quantization.

Also, in order to compute FCN scores for the labels-to-video experiments, an FCN model should be trained on the Viper dataset or at least on Cityscapes. I can't find an FCN model pre-trained on those datasets, but if you have a link for the model you used during the experiments, could you provide the link?

**Comment by wangkaihong** [comment]
Hi,

I just updated the README and included the eval scripts with the FCN model pretrained on Viper for L2V eval. I am not 100% sure they are the ones I used though as I switch to a new server later, so please let me know if they don't work.

**Comment by shadow2496** [comment]
I'll try the scripts. Thank you very much.

**Comment by shadow2496** [comment]
I'm closing the issue cuz I checked the scripts output almost the same values in the paper.
